torch_disc/torch_disc/csrc/disc_compiler/disc_compiler.cpp

```diff
@@ -68,12 +68,18 @@ void EnhancementInputShape(
   }
 }
 
-std::shared_ptr<Executable> CompileToDiscExecutable(
+ExecutablePtr CompileToDiscExecutable(
     const std::shared_ptr<torch::jit::Graph>& graph,
     c10::ArrayRef<torch::lazy::BackendDataPtr> arguments) {
+  bool disable_disc = lazy_tensors::sys_util::GetEnvBool("DISABLE_DISC", false);
+  if (disable_disc) {
```

Comment:
Is it used to testing TorchScript execution results?

Reply:
Yes.